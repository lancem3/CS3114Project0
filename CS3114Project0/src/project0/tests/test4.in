7 5 7
2 1
3 1
1 4
6 7
6 5
1
2
4
5
6
3
7
