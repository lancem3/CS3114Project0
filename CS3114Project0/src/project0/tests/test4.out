2 3 4
1
1
6
7 5
1
6
